### PR TITLE
Feature - Update CreateSitePage to use ui-react

### PR DESCRIPTION
**Pull request by MoiyadKayda**
## Screenshots:
image

## Checklist:

- [ ] I have updated the CHANGELOG with a summary of my changes

### CODE REVIEW

File: src/packages/console/src/App.tsx [hunk 1]
```diff
@@ -105,6 +105,9 @@ const theme = buildTheme({
     error: {
       color: '$colors.error',
     },
+    selectItemText: {
+      'font-size': '14px',
```
Comment (krishan711) [inline src/packages/console/src/App.tsx]: font sizes should be in em

File: src/packages/console/src/pages/createSitePage.tsx [hunk 2]
```diff
@@ -143,26 +139,7 @@ export const CreateSitePage = (): React.ReactElement => {
...
-                        error={selectedAccountIdError !== undefined}
-                        label='Account'
-                        // helperText={selectedAccountIdError}
-                      >
-                        {accounts?.map((account: Account, index: number): React.ReactElement => (
-                          <MenuItem key={index} value={account.accountId}>{account.name}</MenuItem>
-                        ))}
-                      </Select>
-                    </FormControl>
+                    <OptionSelect selectedItemKey={String(selectedAccountId)} inputWrapperVariant={selectedAccountIdError && 'error'} messageText={selectedAccountIdError} onItemClicked={onAccountSelected} options={accounts?.map((account: Account) => ({ itemKey: String(account.accountId), text: account.name, textVariant: 'selectItemText' })) || []} />
```
Comment (krishan711) [inline src/packages/console/src/pages/createSitePage.tsx]: put props on newlines pls

File: src/packages/console/src/App.tsx (hunk 1 above)
Comment (MoiyadKayda) [reply src/packages/console/src/App.tsx]: please check the lint error, it has something to do with the everyview-tracker

File: src/packages/console/src/pages/createSitePage.tsx [hunk 3]
```diff
@@ -143,26 +139,15 @@ export const CreateSitePage = (): React.ReactElement => {
...
-                        {accounts?.map((account: Account, index: number): React.ReactElement => (
-                          <MenuItem key={index} value={account.accountId}>{account.name}</MenuItem>
-                        ))}
-                      </Select>
-                    </FormControl>
+                    <OptionSelect
+                      selectedItemKey={String(selectedAccountId)}
+                      inputWrapperVariant={selectedAccountIdError && 'error'}
+                      messageText={selectedAccountIdError} onItemClicked={onAccountSelected}
+                      options={accounts?.map((account: Account) => ({ itemKey: String(account.accountId),
```
Comment (krishan711) [inline src/packages/console/src/pages/createSitePage.tsx]: rendering should go on a newline as well like:

```
prop={......: React.ReactElement => (
  <Component .... />
)}
```

Comment (krishan711) [reply src/packages/console/src/pages/createSitePage.tsx]: ah this isnt rendering, wierd. will think of a better format just a sec.

Comment (krishan711) [reply src/packages/console/src/pages/createSitePage.tsx]: ah yes use the same format pls (and add typing)

Comment (MoiyadKayda) [reply src/packages/console/src/pages/createSitePage.tsx]: didn't get you

Comment (MoiyadKayda) [reply src/packages/console/src/pages/createSitePage.tsx]: Do you mean the itemKey needs to be on newline ?

Comment (krishan711) [reply src/packages/console/src/pages/createSitePage.tsx]: itemKey and the bracket just before it (not the round bracket, just the curly one)

File: src/packages/core/src/external/everyviewAnalytics.tsx [hunk 4]
```diff
@@ -19,6 +19,7 @@ export const EveryviewAnalytics = (props: IEveryviewAnalyticsProp): null => {
     }
 
     // NOTE(krishan711): figerprintjs references window immediately: https://github.com/fingerprintjs/fingerprintjs/issues/602
+    // eslint-disable-next-line
```
Comment (krishan711) [inline src/packages/core/src/external/everyviewAnalytics.tsx]: you should disable the specific error only (you are currently disabling all errors for the next line

**Review by krishan711 — APPROVED** [review]

**Pull request merged**